packages/client/src/internals/TRPCClient.ts

```diff
@@ -82,7 +82,7 @@ export interface TRPCRequestOptions {
   /**
    * Pass additional context to links
    */
-  context?: OperationContext;
+  requestContext?: OperationContext;
```

Comment:
Could you add a note of this in https://alpha.trpc.io/docs/migrate-from-v9-to-v10 anywhere? I'll clean it up later.

@sachinraja @jlalmes what do you think we should do here now that the `context` keyword is taken?

Some options:

- Put all trpc option under `trpc` - quite annoying for the 99% use-case but also nice to have a distinct place for "our" versus "theirs" in options
- Call it `trpcContext`? Potentially prefix all our vars with `trpc`?
- `operationContext`?
- `opContext`?
- `callContext`?

Reply:
Moved to #2394.

Will merge this one and we can iterate as it's still alpha.